TimeTrackerManager/TimeTrackerManager/AddUserViewController.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  AddUserViewController.swift
+//  TimeTrackerManager
+//
+//  Created by Bocanu Mihai on 15.12.2021.
+//
+
+import UIKit
+
+class AddUserViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        // call the 'keyboardWillShow' function when the view controller receive the notification that a keyboard is going to be shown
```

Comment:
daca ai adaugat Observer aici trebuie sa faci si remove cand dispare ecranul asta altfel e memory leak

Reply:
de fapt sunt eu prea batran, nu mai trebuie din iOS 9 sa faci asta manual. https://useyourloaf.com/blog/unregistering-nsnotificationcenter-observers-in-ios-9/

Reply:
:))